app/policies/data_export_policy.rb

```diff
@@ -1,9 +1,17 @@
 class DataExportPolicy < ApplicationPolicy
+  def index?
+    division_admin(division: Division.root)
+  end
+  
```

Comment:
To pass the specs below, I only need to define ` index?`  here. I put the other actions in too so the intention is visible in the policy, and they don't depend on other code. The other specs pass without the code because Application Policy has `show`, `create`, and `new` all checking `division_member_or_admin` and the data_export factory has the division set to root_division.